docs/upgrade.txt

```diff
@@ -79,6 +79,15 @@ This library version introduces the following breaking changes:
   of this behavior, you cannot have two separate ``id`` and ``_id`` fields in your
   documents.
 
+- Removes support for the ``$collection`` property and associated methods.
+  The following list contains the removed methods and provides alternative methods
+  that you can use to replace the functionality:
+  
+  - ``Model::collection()``: use ``Model::table()``
+  - ``DB::collection()``: use ``DB::table()``
+  - ``Connection::collection()``: use ``Connection::table()``
+  - ``Schema::collection()``: use ``Schema::table()``
```

Comment:
Actually there is only 2 methods removed in `MongoDB\Laravel\Connection` and `MongoDB\Laravel\Schema\Builder`, where they can be used as instance method.

But they are exposed in the Laravel facades `Illuminate\Support\Facades\DB` and `Illuminate\Support\Facades`, which expose them as static method. (Yes, this is very confusing).

```suggestion
  - ``DB::collection()``: use ``DB::table()``
  - ``Schema::collection()``: use ``Schema::table()``
```

Reply:
Okay I see, updated! Should I mention the static methods here?